src/types/common.d.ts

```diff
@@ -18,6 +46,13 @@ export interface KeyProps {
   displaced: boolean;
 }
 
+export interface Tempo {
```

Comment:
This should be `StaveTempoOptions` and belongs in `stavetempo.ts`

Reply:
done